### PR TITLE
Sqlbase

### PR DESCRIPTION
**Pull request by JohnOldman2018**
The final script with corrections.

### CODE REVIEW

File: src/main/resources/SQL/tables.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE Countries (
+    code VARCHAR (2) PRIMARY KEY,
+    name VARCHAR (70) NOT NULL
+);
+
+CREATE TABLE Rooms (
+    room_ID INTEGER (8) PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    room_name VARCHAR (150) NOT NULL,
+    country_code VARCHAR (150) NOT NULL,
```
Comment (amussagaliyev) [inline src/main/resources/SQL/tables.sql]: country_code's length should be 2. It is a code from Country table

File: src/main/resources/SQL/tables.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE Countries (
```
Comment (amussagaliyev) [inline src/main/resources/SQL/tables.sql]: Let's use lowercase for table and column names.

File: src/main/resources/SQL/tables.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    code VARCHAR (2) PRIMARY KEY,
+    name VARCHAR (70) NOT NULL
+);
+
+CREATE TABLE Rooms (
+    room_ID INTEGER (8) PRIMARY KEY NOT NULL AUTO_INCREMENT,
+    room_name VARCHAR (150) NOT NULL,
+    country_code VARCHAR (150) NOT NULL,
+    light_is_on BIT DEFAULT 0 NOT NULL
```
Comment (amussagaliyev) [inline src/main/resources/SQL/tables.sql]: let's use tinyint for boolean column.
Please read on
https://www.javatpoint.com/mysql-boolean#:~:text=MySQL%20does%20not%20contain%20built,to%200%20and%201%20value.

**Review by amussagaliyev — CHANGES_REQUESTED** [review]

File: src/main/resources/SQL/tables.sql (hunk 3 above)
Comment (JohnOldman2018) [reply src/main/resources/SQL/tables.sql]: Does that mean we will use  IF(pass, 'true', 'false')  in our queries to DB?

Comment (JohnOldman2018) [reply src/main/resources/SQL/tables.sql]: As I got it, IF(pass, 'true', 'false')  work in DB the same as ternary operator  ? true: false; ?

**Pull request closed without merging**